### PR TITLE
style(typography): added typography margin styles

### PR DESCRIPTION
**Pull request by paulAcato**
Added styling for the typography for following up elements since it took the default browser margins.

### CODE REVIEW

File: components/Typography/src/paragraph.scss [hunk 1]
```diff
@@ -29,3 +31,25 @@
   line-height: var(--denhaag-posttypelabel-line-height, 1.5);
   text-transform: var(--denhaag-posttypelabel-text-transform, uppercase);
 }
+
+// When following up specific elements.
```
Comment (robbieatacato) [inline components/Typography/src/paragraph.scss:35]: Comment can be removed.

File: components/UnorderedList/src/index.scss [hunk 2]
```diff
@@ -5,6 +5,17 @@
   @extend .utrecht-unordered-list;
 
   color: var(--denhaag-unordered-list-color);
+  margin-block-start: var(--denhaag-unordered-list-margin-block-start, 0);
+}
+
+.utrecht-paragraph + .denhaag-unordered-list {
+  // 16
```
Comment (robbieatacato) [inline components/UnorderedList/src/index.scss:12]: Comment can be removed.

File: components/UnorderedList/src/index.scss [hunk 3]
```diff
@@ -5,6 +5,17 @@
...
+  --denhaag-unordered-list-margin-block-start: var(--denhaag-unordered-list-paragraph-margin-block-start);
+}
+
+.utrecht-paragraph--lead + .denhaag-unordered-list {
+  // 32
```
Comment (robbieatacato) [inline components/UnorderedList/src/index.scss:17]: idem

File: components/UnorderedList/src/stories/bem.stories.mdx [hunk 4]
```diff
@@ -47,3 +47,22 @@ import "../index.scss";
   </Story>
   {/* eslint-enable react/no-unknown-property */}
 </Canvas>
+
+### Unordered list + Paragraph
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Unordered list + Paragraph">
+    <ul class="denhaag-ordered-list">
```
Comment (robbieatacato) [inline components/UnorderedList/src/stories/bem.stories.mdx:56]: I think you meant: `<ul class="denhaag-unordered-list"> `
And if this is not the situation, than you still can use this class extending it with `@extend .denhaag-ordered-list` in the scss file.

**Review by robbieatacato — CHANGES_REQUESTED** [review]

File: components/UnorderedList/src/stories/bem.stories.mdx (hunk 4 above)
Comment (paulAcato) [reply components/UnorderedList/src/stories/bem.stories.mdx:56]: Changed the `denhaag-ordered-list` to `denhaag-unordered-list`. and the styles are already extended ;-)

**Review by robbieatacato — APPROVED** [review]

**Pull request merged**